mols2grid/molgrid.py

```diff
@@ -802,6 +808,36 @@ def get_selection(self):
             columns=self._extra_columns
         )
 
+    def get_selection_state(self):
```

Comment:
since this is specific to marimo, it should have a more descriptive name, something like `marimo_selection_getter`?